tests/integration/tests/test_mender_configure.py

```diff
@@ -67,12 +218,38 @@ def test_mender_configure(setup_test_container, setup_tester_ssh_connection):
             remote_path="/data/configuration-artifact.mender",
         )
 
+        # Disable writes to the file
+        run(
+            setup_tester_ssh_connection,
+            "chattr -i /var/lib/mender-configure/device-config.json",
```

Comment:
This would be great as a separate test, but isn't it better to allow this write, since you are already returning 2 in the apply script? The script is supposed to roll this back on its own.

Reply:
Ah, this is a left-over. Unfortunately, I cannot use chattr because it's a tmpfs, I removed it.